docs/source/_toctree.yml

```
@@ -6,6 +6,8 @@
     title: Quicktour
   - local: install
     title: Installation
+  - local: common_issues
+    title: Common issues
```

I think this sounds stronger/more actionable than "Common issues". I would also place this section in the Developer Guides since I don't think it's good to present issues/troubleshooting at the beginning of the user's journey.

```suggestion
    title: Troubleshoot
```